lib/rpc/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { ChainId } from '@uniswap/sdk-core'
+
+export function chainIdToNetworkName(networkId: ChainId): string {
+  switch (networkId) {
```

Comment:
does `SingleJsonRpcProvider` not accept the `ChainId` object? this seems redundant but tell me if i am missing something here?